department-of-veterans-affairs/vets-website
# Pension: spouse's former marriage

## Summary

On Step 4 of Household information,
on page for Spouse's former marriage,
- Add blue info alert before the first question/field on the page
- ensure all the questions are in the order as shown on the designs
- ensure all the questions are worded the same as the designs
- add new "Other" option for: How did the marriage end?
- add new multi-spouse entry option
as shown on the designs.

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the pension_form_enabled flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run vets-website and vets-api

## How to verify
1. Go to http://localhost:3001/pension/application/527EZ in your browser
2. Select 'Start your application without signing in'
3. Proceed to Chapter 4 by filling out the required fields on Chapter 1, Chapter 2, and 3. 
4. Answer 'married' or 'separated' for Marital Status and fill out the next pages until you reach the 'Spouse's former marriage' page.
5. You should not be able to proceed or add an additional marriage without all required fields being filled.
6. You should be able to add multiple marriages. 

## Related issue(s)

- [Link to GitHub Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69963)
- [Link to GitHub Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/71650)
- [Link to Epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done


## Screenshots



| Before | After (Individual) | After (Multi) | After (Validation) |
| ------- | ----------------|--------------|------------------|
| <img width="264" alt="Screenshot 2023-12-07 at 5 13 35 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/053245c6-c1e7-4048-a843-de8dccf57435"> | <img width="303" alt="after1" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/598b3ec5-4c5c-4060-b169-096a6cc8ec50"> | <img width="302" alt="after2" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/c2d15e34-203f-440f-aca5-e3d78c262e6a"> | <img width="234" alt="after-validation" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/2bf21362-879d-44e1-acde-f6fef6ae93f9"> |

## What areas of the site does it impact?

Pension Benefits

## Acceptance criteria

- [Link to GitHub Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69963)

### Quality Assurance & Testing


### Error Handling


### Authentication
